Sources/MessagingInApp/Provider/InAppProvider.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 // wrapper around Gist SDK to make it mockable
 protocol InAppProvider: AutoMockable {
-    func initialize(siteId: String, region: Region, delegate: GistDelegate)
+    func initialize(siteId: String, region: Region, delegate: GistDelegate, logging: Bool)
```

Comment:
Do you plan to update the test cases for this change in a separate PR ?

Reply:
We don't have a test for initialization for `MessagingInApp`, I assume when we add that then we can check for logging too.